sympy/utilities/lambdify.py

```diff
@@ -8,7 +8,12 @@
 from functools import wraps
 import inspect
 import textwrap
-
+from sympy import *
+# from sympy import Derivative
```

Comment:
Why is this commented-out code here?

Reply:
It was by mistake by me, it won't happen again.